suc/pkg/cmd.go

```diff
@@ -0,0 +1,64 @@
+package pkg
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/rancher/wins/suc/pkg/rancher"
+	"github.com/rancher/wins/suc/pkg/service"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+func Run(_ *cli.Context) error {
+	var errs []error
+	refreshService := false
+
+	initialState, err := service.BuildInitialState()
+	if err != nil {
+		return fmt.Errorf("could not build initial state for rancher-wins: %v", err)
+	}
+
+	logrus.Infof("Updating rancher connection info")
+	o, err := rancher.UpdateConnectionInformation()
+	if err != nil {
+		logrus.Errorf("Could not update rancher connection information. Script output:")
+		logrus.Error(o)
+		return fmt.Errorf("error encountered while refreshing connection information: %v", err)
+	}
+
+	logrus.Infof("successfully updated connection info")
+	if o != "" {
+		logrus.Infof(" Script output:\n%s", o)
+	}
+
+	// update the config using env vars
+	restartServiceDueToConfigChange, err := service.UpdateConfigFromEnvVars("")
+	refreshService = restartServiceDueToConfigChange
+	if err != nil {
+		errs = append(errs, err)
+	}
+
+	if refreshService && (errs == nil || len(errs) == 0) {
```

Comment:
`errs` has been defined as `[]error` so it could not be nil.

Reply:
I've removed the use of an error slice here, it made more sense in my POC for this change but is no longer needed